Update the Setting Up a Gift Registry columns (#1339)
magento/merchdocs: src/marketing/gift-registry-create.md

|Country|Displays the attribute as a drop-down list of countries. Set **Show Region** to: Yes / No.|
|**_Static Types_**| |
|Event Date|Determines how the date attribute is used in the store. Options: <br/>**Searchable** - Determines if the attribute is available for Advanced Search. Options: Yes / No.<br/>**Is Listed** - Determines if the event is included in the list of events that is available in the store. Options: Yes / No. <br/>**Date Format** - Determines the format of the event date. Options: Short (3/23/2014) / Medium (Mar 23, 1914) / Long (March 23, 1914) / Full (Sunday, March 23, 2014)|
|Event Country|Displays the attribute as a drop-down list of countries. Options: <br/>**Searchable** - Determines if the attribute is available for Advanced Search. Options: Yes / No.<br/>**Is Listed** - Determines if the event is included in the list of events that is available in the store. Options: Yes / No. <br/>**Show Region** - Determines the region of the event.|
|Event Location|The location of the event that is related to the gift registry. <br/>Set **Is Searcheable** to: Yes / No <br/>Set **Is Listed** to: Yes / No|

> |Role|The role that identifies who the gift is for. <br/>Set **Is Searcheable** to: Yes / No <br/>Set **Is Listed** to: Yes / No <br/>Click **Add New Option** to add more conditions to the drop-down menu:<br/>**Code** - A unique name to identify the attribute internally.<br/>**Label** - The name that identifies the attribute in the customer’s account dashboard.<br/>**Is Default** - Set the switch to select the default condition.<br/>**Delete Option** - To delete the option, click **Delete Option**.<br/>For example, `Bride`, `Groom`, or `Partner`.|

Reviewer: ```suggestion
|Role|The role that identifies the gift recipient. For example, `Bride`, `Groom`, or `Partner`.<br/>**Is Searcheable** - Set to `Yes `/ `No` <br/>**Is Listed** - Set to `Yes` / `No` <br/>**Add New Option** - Click  to add more conditions to the drop-down menu:<br/>**Code** - A unique name to identify the attribute internally.<br/>**Label** - The name that identifies the attribute in the customer’s account dashboard.<br/>**Is Default** - Set this switch to select the default condition.<br/>**Delete Option** - Click to delete the option.|
```

Author: @leslietilling Thank you, done  ☑️
